server/models/study.py

```diff
@@ -280,6 +341,9 @@ def filter(self, study, user=None, additionalKeys=None):
             }
         }
 
+        if User().canAdminStudy(user):
+            output['waitingUsers'] = getSortedUserList(Study().getWaitingUsers(study))
```

Comment:
To eventually accommodate a feature where admins issued soft "invites" for users to participate in a study as annotators, I think this should be given a more precise name than `waitingUsers`. I can't think of a good one right now though.

Reply:
How about `requests`? This avoids using a long name and matches a similar field in Girder's group model.

Reply:
Honestly, that seems a little ambiguous still, since an admin could also issue "requests" for users to join. I'm fine with longer / compound names, I don't see big advantages to being overly terse.

There are only two hard problems in computer science...